echo-of-the-song/Assets/Game/Scripts/Damage/IDetector.cs

```diff
@@ -0,0 +1,10 @@
+using Mechanics.Damage;
+using System.Collections.Generic;
+
+namespace Mechanics.DetectorCollision
+{
+    public interface IDetector
```

Comment:
What doed detector hav to detect?
Example : 
public interface IDetector<T> where T : Object
.......
public DetectionData<T> Detect();